lib/motors/can_messages.cpp

```diff
@@ -0,0 +1,64 @@
+#include "can_messages.hpp"
+
+namespace hyped::motors {
+
+CanMessages::CanMessages(std::shared_ptr<io::ICan> can) : can_(can)
```

Comment:
very nitpicky, but `can_(std::move(can))` is supposedly more efficient and has been enforced so far